src/meta/src/controller/catalog.rs

```diff
@@ -3261,7 +3261,27 @@ impl CatalogControllerInner {
             )
             .all(&self.db)
             .await?;
-        // TODO: filter out inner connector source that are still under creating.
+
+        // filter out inner connector sources that are still under creating.
+        let created_table_ids: HashSet<TableId> = Table::find()
+            .select_only()
+            .column(table::Column::TableId)
+            .join(JoinType::InnerJoin, table::Relation::Object1.def())
```

Comment:
Is object1 a typo? Should it be Object

Reply:
Nope, `Table` has multiple relations referring to `Object` and the name is generated by sea-orm.